lib/streamlit/runtime/runtime.py

```diff
@@ -490,15 +501,8 @@ async def _loop_coroutine(
             else:
                 raise RuntimeError(f"Bad Runtime state at start: {self._state}")
 
-            # Store the eventloop we're running on so that we can schedule
-            # callbacks on it when necessary. (We can't just call
-            # `asyncio.get_running_loop()` whenever we like, because we have
-            # some functions, e.g. `stop`, that can be called from other
-            # threads, and `asyncio.get_running_loop()` is thread-specific.)
-            self._eventloop = asyncio.get_running_loop()
```

Comment:
This was accidentally left in from a previous commit, I think. `self._eventloop` isn't used anywhere (and isn't declared on the constructor.)